src/shared/components/RichTable.scss

```diff
@@ -32,6 +39,10 @@
   &_table {
     table-layout: fixed;
     margin-bottom: 0;
+
+    @include media-breakpoint-down(sm) {
+      width: auto;
```

Comment:
This makes tables with only a few columns not take up all the space on small screen sizes:
![screenshot](https://user-images.githubusercontent.com/7670245/82739121-03130280-9d0b-11ea-87f2-9aacb399fa9a.png)

Tried setting it to 100%, but then tables with lots of columns get crushed and all the text has to break. Think in the future, we might have to specify "important columns" and then drop the rest on small screen sizes.

Reply:
Fixed in 27ad7b3